airflow/cli/cli_parser.py

```diff
@@ -468,7 +468,18 @@ def string_lower_type(val):
     help="Don't preserve purged records in an archive table.",
     action="store_true",
 )
-
+ARG_DB_EXPORT_TO_CSV = Arg(
```

Comment:
I feel like we should merge this with `output-path`, which should be required.

Reply:
or what if we have output path and format. we could support only csv at the moment but json down the road.
perhaps output path can optionally be `-` for stdout.

Reply:
Maybe we shouldn't combine it but have e.g `--export-to` that defaults to `database` with other possible options like `csv`, `json` etc? `csv` at the moment of course.

Reply:
That works for me. Json really appeals to me since you could jam it all in a single file, which also makes stdout much easier.

Reply:
I poked around `json` and there are issues on serialization. I think archiving to `json` should be done in a separate PR

Reply:
Yeah i am not suggesting you implement json now... only suggesting that we have separate "out file" from "format" params